### ISSUE TITLE
Translate 5 Medium blog posts to Italian

### ISSUE THREAD

**Issue by akhs13** [rchain/bounties#790]
### Benefit to RChain

Translation of the RChain Medium blog posts to other languages means good global marketing.

### Budget and Objective

_Estimated Budget of Task:_ **$[1950]**
_Estimated Timeline Required to Complete the Task:_ **[7 days]**
_How will we measure completion?_ **Post "RNode v0.4.1 introduces sending Rholang smart contracts across the network" - "RChain Weekly Update (Debrief 81) — 13 June, 2018" - "RChain Weekly Update (Debrief 80) June 6-–2018"-"RChain Weekly Update (Debrief 82) — 20 June, 2018" -"The Future of Blockchain Event — with Greg Meredith and Greg Heuss" Medium blog posts translations to the [RChain Cooperative-Italian](https://medium.com/@rchaincoop.it)**

### Legal

_Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws._

### Translation Checklist

*for non translation issues, delete this section*

  - Why translate this document?
    - It's important to translate because ____
    - [x] It's an [RChain blog](https://medium.com/rchain-cooperative) item
    - [ ] It's a document on https://developer.rchain.coop/ that has been endorsed
          for translation by @Jake-Gillberg, @MParlikar, or @KellyAtPyrofex 
  - How can people who do not speak this language be assured of the quality of the translation? (all are required)
    - [x] several reviewers are involved (_assign the ticket to them_)
    - [x] at least two of them have high proficiency in _the target language_
    - [x] at least two of them have high proficiency in English
    - [x] at least two of them have either
       - [x] formal training: degree / certificate: CELI 5 from institution of Perugia
       - [x] a track record of quality work on rchain bounties: #645 #722 #717 #691 #687 #684 #683 #670 #590 #587 #580 # #577 #576 #574 #537 #502 #489 #488 #480 #369 #368 #367
       - [ ] another body of work readily available for community review: _____
  - What about maintenance? (choose one)
    - [x] the source document is stable; further translation work is not expected
    - [ ] our plan to keep the translation up to date is: ____

**Comment by allancto** [comment]
@akhs13 @Chris-Arv translation per se is NOT a priority for our Cooperative. Giving back to our community (opinions, local wisdom, research, development based on documents which you might need to be translated)- these are. 

Here is a document which has been started explaining the kind of "Value Added Translation" that our Cooperative may find useful. Please consider contributing to this document. If these ideas make sense to you you may also want to consider developing the ideas further, and perhaps helping us a translation "Guide". 

Thanks!
@allancto @michaelizer @zero-andreou @ICA3DaR5 

https://docs.google.com/document/d/1wh-GYj_MxquhUL-VyE0BkeuoskHHeq5KVdd8tL77AKQ/edit?usp=sharing
There are 1001 ways to add value. When you are reading and understanding something you will ALWAYS have value to add. We in the bounty system are trying to create as much value for our Cooperative as we are able. Translation is important because it stimulates diversity and discussion. 

Value added translation is about giving back to the source community. It may take the form of praise or criticism, it may be research that you did while trying to understand the material, it may be simply a summary of what you saw in the material, expressed in your own words not in the original.

**Comment by akhs13** [comment]
I will add value to the translation. @allancto

**Comment by deannald** [comment]
For future translation work, I'm going to defer to @patrick727 and the marketing team to prioritize which documents RChain Cooperative is looking for translation (that's not to say you can't propose others, of course). The Task Approval Committee and RChain Cooperative are trying to ensure that high valued articles, blog posts and other materials are given top priority and, further, are distributed in a manner that has great reach.

I'd like to get weigh in from @patrick727 on value of this particular issue so that budget allotted/voted is commensurate with value (higher priority translations = higher budget).

@dckc @ian-bloom

**Comment by akhs13** [comment]
It is important for every member of the community to read these blogs and be informed about what is happening in the hangouts. By reading these blogs you can get information on the deals with the companies that collaborate with the Rchain.Also you can find out how each team works and what is the purpose of each team(for example Rholang team,Roscala team,Node team). Furthermore it is important for each member to know the updates that happen on the platform.This way you can anwser various questions that occur about the validation and convertion from Rhoc to Rev. Moreover you can find several references  about the technologies that are being developed on the new platform. Translating these blogs is very helpful for foreign members to form their own perspective and to trigger new members to get involved.

**Comment by dckc** [comment]
I agree that written accounts of the weekly update are valuable to spread far and wide.

My one suggestion is that we make better use of headlines. See [Nielsen on Microcontent](https://www.nngroup.com/articles/microcontent-how-to-write-headlines-page-titles-and-subject-lines/). _This suggestion probably belongs elsewhere, but I'd appreciate it if the folks involved here would excuse me for expanding the scope of your work a bit._

**Comment by deannald** [comment]
I would like to learn a little more of the added value you are planning on providing. As with all translations in the future, we'd like to see something beyond what can be acquired using Google translate. So, if time and money are being spent on translation, it must go beyond what can easily be achieved by right-click and translate to "language of your choice". Especially if the output has limited readership.
